enterprise/frontend/src/metabase-enterprise/data_editing/tables/types.ts

```diff
@@ -32,3 +33,12 @@ export type UpdatedRowCellsHandlerParams = {
   data: Record<DatasetColumn["name"], RowValue>;
   rowIndex: number;
 };
+
+export type FieldWithMetadata = Field & {
```

Comment:
@deniskaber I couldn't find typings for these properties. Should we keep it here or move somewhere else?

Reply:
@deniskaber I've added additional properties to the `Field` type in 85a5def271